### PR TITLE
Create indexes for all foreign key columns

### PR DESCRIPTION
**Pull request by tung2744**
ref #3991 #3984

### CODE REVIEW

**Comment by tung2744** [comment]
Pushed a commit to add indexes mentioned in https://github.com/authgear/authgear-server/issues/3984

File: migrations/authgear/20240318191059-add_indexes_to_foreign_key_columns.sql [hunk 1]
```diff
@@ -12,6 +12,10 @@ CREATE INDEX _auth_user_role_user_id ON _auth_user_role (user_id);
 CREATE INDEX _auth_user_role_role_id ON _auth_user_role (role_id);
 CREATE INDEX _auth_verified_claim_user_id ON _auth_verified_claim (user_id);
 
+CREATE INDEX _auth_user_app_id ON _auth_user (app_id);
+CREATE INDEX _auth_user_app_id_created_at ON _auth_user (app_id, created_at);
```
Comment (louischan-oursky) [inline migrations/authgear/20240318191059-add_indexes_to_foreign_key_columns.sql:16]: The default is ASC NULLS LAST. https://www.postgresql.org/docs/current/indexes-ordering.html
But our usage is DESC NULLS LAST.

File: migrations/authgear/20240318191059-add_indexes_to_foreign_key_columns.sql [hunk 2]
```diff
@@ -12,6 +12,10 @@ CREATE INDEX _auth_user_role_user_id ON _auth_user_role (user_id);
...
+CREATE INDEX _auth_user_app_id_last_login_at ON _auth_user (app_id, last_login_at);
```
Comment (louischan-oursky) [inline migrations/authgear/20240318191059-add_indexes_to_foreign_key_columns.sql:17]: Same here.

**Comment by tung2744** [comment]
@louischan-oursky  Updated, thanks!

**Pull request merged**